lib/presentation/widget/shared_space/file_surfing/workgroup_nodes_surfing_viewmodel.dart

```diff
@@ -115,6 +123,8 @@ class WorkGroupNodesSurfingViewModel extends BaseViewModel {
               loadAllChildNodes();
             } else if (success is CreateSharedSpaceFolderViewState) {
               loadAllChildNodes();
+            } else {
+              loadSorterAndAllChildNodes();
```

Comment:
Hmm, this is too risky, you should specify exactly which `success` we should execute `loadSorterAndAllChildNodes `